src/reducers/folder.js

```diff
@@ -20,6 +20,8 @@ export const files = (state = [], action) => {
         ...state,
         action.file
       ]
+    case TRASH_FILE_SUCCESS:
+      return state.map(file => file.id === action.file.id ? action.file : file)
```

Comment:
If `id` are the same we return `action.file` but if not we return `file`. Why don't we always return `file`?
Is it possible that `action.file !== file` even when `action.file.id === file.id`?